src/test/kotlin/no/nav/amt/person/service/integration/kafka/utils/KafkaMessageSender.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.amt.person.service.integration.kafka.utils
+
+import no.nav.amt.person.service.kafka.config.KafkaProperties
+import no.nav.common.kafka.producer.KafkaProducerClientImpl
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Component
+import java.util.*
+
+@Component
+class KafkaMessageSender(
+	properties: KafkaProperties,
+	@Value("\${app.env.endringPaaBrukerTopic}")
+	private val endringPaaBrukerTopic: String,
+) {
+	private val kafkaProducer = KafkaProducerClientImpl<ByteArray, ByteArray>(properties.producer())
+
+	fun sendTilEndringPaaBrukerTopic(jsonString: String) {
+		kafkaProducer.send(
+			ProducerRecord(
+				endringPaaBrukerTopic,
+				UUID.randomUUID().toString().toByteArray(),
```

Comment:
Den topicen du skal lese fra, har ikke den key/value som er string? Ser sånn ut i oppsettet lenger opp :)

Reply:
Den har det, men jeg har aldri sett ikke noe forskjell på ByteArray og String i tidligere tester. Konsumeren klarer vel å konvertere byte til string selv, men det er mulig det er noen UTF-8 feller med det?

Men grunnen til at denne er ByteArray, er fordi den er mer generisk pluss at jeg jeg yoinka den fra amt-tiltak. Vi kommer til å måtte teste topics som bruker Avro også, da må vi vel ha ByteArray? Men det går kanskje med flere producere her også?

Reply:
Det er litt rart at det funker, for String og ByteArray ser jo ganske forskjellige ut 😅 Men hvis det virker så virker det 🤷‍♀️ 

Med avro må du ha en egen avroserializer/deserializer. Ja, du kan ha flere producere, Magnus satte opp det i amt-tiltak :)